src/build/common/steps/nuget-license-check.steps.yaml

```diff
@@ -0,0 +1,39 @@
+# Job template to deploy to a specific environment
+parameters:
+  projectPaths: 
+    default: []
+
+steps:
+  - script: |
+      dotnet new tool-manifest
+      dotnet tool install dotnet-project-licenses
+    name: ${{ parameters.name }}
+    displayName: Install license check dotnet tool
+  - task: PowerShell@1
+    displayName: "Process Allowed Licenses and Mappings"
+    condition: and(succeeded(), ne(variables['parameters.projectPaths'], ''))
+    inputs:
+      scriptType: inlineScript
+      inlineScript: |
+        $licenses = @("MIT","Apache-2.0","BSD-3-Clause","Apache-2.0 OR MPL-2.0","MS-EULA","X11","Unlicense","0BSD","BSD-2-Clause")
```

Comment:
There definitely doesn't need to be any project specific overrides of any of these configs? Or at least not in v1?